.jenkins/common.groovy

```diff
@@ -40,9 +39,6 @@ def runCompileCommand(platform, project, jobName, boolean debug=false)
             cmake -DCMAKE_BUILD_TYPE=${buildType} -DCMAKE_CXX_COMPILER=${compiler} -DCODE_OBJECT_VERSION=${cov} -DTensile_ROOT=\$(pwd)/../Tensile ../HostLibraryTests
             make -j\$(nproc)
 
-            popd
```

Comment:
re-add this to get rid of error when publishing results

Reply:
I am just getting rid of publish results, we do not have that file.